python/vowpalwabbit/pyvw.py

```diff
@@ -1184,6 +1184,43 @@ def __str__(self):
         )
 
 
+class cb_continuous_label(abstract_label):
```

Comment:
We should conform to PEP8 for newly added code and move towards it for old code
```suggestion
class CBContinuousLabel(abstract_label):
```